TwitchChannelPointsMiner/classes/Twitch.py

```diff
@@ -50,6 +63,18 @@ def __init__(self, username, user_agent, password=None):
             CLIENT_ID, username, self.user_agent, password=password
         )
         self.running = True
+        self.device_id = ''.join(
+            random.choices(string.ascii_letters + string.digits, k=26)
```

Comment:
This should be 32 characters long, instead of 26.  

I also suggest using `secrets.choice` to generate these instead of `random.choice` (as is done when generating transaction IDs), to avoid the heightened chance of collisions with other clients. 

For `self.client_session`, you can use `secrets.token_hex(16)`

Reply:
Done. Thanks for suggestions.